integration/internal/scheduler.go

```diff
@@ -0,0 +1,15 @@
+package internal
+
+var SchedulerDefaultArgs = []string{
```

Comment:
I'm wondering if these could be made public. So that to allow people to append if they want, or not, etc.

Reply:
We made all the default arguments public.